src/inference/Inference.jl

```diff
@@ -314,10 +316,13 @@ function AbstractMCMC.bundle_samples(
     N::Integer,
     ts::Vector,
     chain_type::Type{Chains};
+    raw_output::Bool=false,
```

Comment:
I don't think this is needed? That's the default in AbstractMCMC if you specify `chain_type = Any`:
https://github.com/TuringLang/AbstractMCMC.jl/blob/dc5a76f45acc3f3b1025ea5856808edc3f07c10a/src/AbstractMCMC.jl#L208-L218

Reply:
```suggestion
```